src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -217,6 +218,7 @@ def __init__(self):
                 "retry_job": self._retry_jobs,
                 "job_logs": self._get_job_logs,
             }
+        self._lookup_all_job_states(ignore_refresh_flag=True)
```

Comment:
Start the jobcomm by getting all job states. Sending them over the websocket once will catch everything, back and front, up to date.